src/shared/utils/ast/extractors.ts

```diff
@@ -84,6 +56,8 @@ function rangeTimes(ast: any, rangeNode: CallExpression): [number, number] {
   return [start, end]
 }
 
+// FIXME: this should not use lookupVariable(ast, value.name)
+// instead should lookup value in scope map
```

Comment:
cannot fix all variable handling in this PR.
comment left for future reference/work.

Reply:
ah...I actually ended up fixing this. Because I needed to.